<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoLang::GoToolsSettingsPageWidget</class>
 <widget class="QWidget" name="GoLang::GoToolsSettingsPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="goImportsCheckBox">
     <property name="text">
      <string>Exexute go&amp;imports on save</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="goFmtCheckBox">
     <property name="text">
      <string>Exexute go&amp;fmt on save</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>229</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
